Source/ExecutionEngine/ExecutionEngine.cs

```diff
@@ -1244,6 +1244,13 @@ private static void CleanupCheckers(string requestId)
             verificationResult.Errors = null;
             verificationResult.Outcome = VCGen.Outcome.Inconclusive;
           }
+          catch(AggregateException e)
```

Comment:
Can we catch something more specific? I wouldn't want to swallow any bugs here but catching a bad solver argument seems good.